realtime_tools/include/realtime_tools/realtime_thread_safe_box.hpp

```diff
@@ -310,7 +310,7 @@ using RealtimeThreadSafeBoxRecursive = RealtimeThreadSafeBoxBase<T, std::recursi
 
 // This is the specialisation we recommend to use in the end
 template <typename T>
-using RealtimeThreadSafeBox = RealtimeThreadSafeBoxStandard<T>;
+using RealtimeThreadSafeBox = RealtimeThreadSafeBoxBase<T, DEFAULT_MUTEX>;
```

Comment:
we could also rename the RealtimeThreadSafeBoxBase to RealtimeThreadSafeBox, because there is no justification of the Base class now?